pkg/testing/integration_register.go

```diff
@@ -279,7 +279,7 @@ var _ = Suite(&MysqlDBDepConfig{
 	},
 })
 
-// MongoDB deployed on openshift cluster
+// MongoDBDepConfig deploys MongoDB on openshift cluster
```

Comment:
It's just a type, it doesn't deploy the app.

Reply:
done